File: trunk/C#/PokerProtocol/Commands/Lobby/AbstractLobbyCommand.cs
using System;
using System.Collections.Generic;
using System.Text;
using EricUtility.Networking.Commands;

namespace PokerProtocol.Commands.Lobby
{
    public abstract class AbstractLobbyCommand : AbstractCommand
    {
        public static new char Delimitter { get { return '|'; } }

        protected override void Append<T>(StringBuilder sb, T thing)
        {
            sb.Append(thing);
            sb.Append(AbstractLobbyCommand.Delimitter);
        }
    }
}

File: JAVA/PokerProtocol/src/bluffinmuffin/protocol/commands/lobby/AbstractLobbyCommand.java
package bluffinmuffin.protocol.commands.lobby;

import bluffinmuffin.protocol.commands.AbstractCommand;

public abstract class AbstractLobbyCommand extends AbstractCommand
{
    public static final char Delimitter = '|';
    
    @Override
    protected <T> void append(StringBuilder sb, T thing)
    {
        sb.append(thing);
        sb.append(AbstractLobbyCommand.Delimitter);
    }
}
